core/state/database.go

```diff
@@ -739,10 +718,9 @@ func (tds *TrieDbState) UnwindTo(ctx context.Context, blockNr uint64) error {
 			}
 		} else {
 			//todo is aggregateBuffer collect data from one block?
-			value, err := account.Encode(ctx)
-			if err != nil {
-				return err
-			}
+			valueLen := account.EncodingLengthForStorage()
+			value := make([]byte, valueLen)
+			account.EncodeForStorage(value)
```

Comment:
Could we incapsulate this 3 lines into single accounts package function or method?

Reply:
Perhaps, as a utility. I expect that in many cases we can use some kind of reusable buffers (there is one such usage in trie.Resolver), that is why I wanted to keep these separate

Reply:
I was thinking of incapsulating such buffers in Account type and use Finalizer to return accolated buffers into the pool.